engine/apps/slack/scenarios/resolution_note.py

```diff
@@ -76,16 +77,27 @@ def process_scenario(
 
         warning_text = "Unable to add this message to resolution note, this command works only in incident threads."
 
+        thread_ts = payload.get("message", {}).get("thread_ts")
```

Comment:
Could you please add a comment that it checks if message is a thread message?

Reply:
done in https://github.com/grafana/oncall/pull/4518/commits/caafcac76edf4239dd43dd19ad15d1f99a341a45